internal/osbuildexecutor/runner-impl-aws-ec2.go

```diff
@@ -53,52 +95,133 @@ func (ec2e *awsEC2Executor) RunOSBuild(manifest []byte, store, outputDirectory s
 		}
 	}()
 
-	logrus.Info("Spinning up jobsite manager")
-	args := []string{
-		"--builder-host",
-		*si.Instance.PrivateIpAddress,
-		"--store",
-		store,
+	ctx, cancel := context.WithTimeout(context.Background(), time.Minute*10)
+	defer cancel()
+	if !waitForSI(ctx, *si.Instance.PrivateIpAddress) {
+		return nil, err
 	}
+	controlFile, err := os.Open(filepath.Join("/var/tmp/osbuild-composer", "control.json"))
```

Comment:
It might be slightly shorter to use `os.WriteFile()` here, something like:
```go
	tmpdir := os.MkdirTemp("/var/tmp/osbuild-composer", "executor-")
	if err != nil {
		return nil, fmt.Errorf("cannot create tempdir for oaas run: %w", err)
	}
	defer os.RemoveAll(tmpdir)
	
	controlFilePath := filepath.Join(tmpdir, "control.json")
	if err = os.WriteFile(controlFilePath, []byte(fmt.Sprintf("{\"exports\": [\"%s\"]}", exports[0])), 0644); err != nil {
		return nil, fmt.Errorf("cannot write control file: %w", err)
	}
```